test/studies/shootout/mandelbrot/ldelaney/PERFEXECOPTS

```diff
@@ -0,0 +1 @@
+--n=16000
```

Comment:
This comment is on the performance execution options for the mandelbrot test. Is it correct to assume that you are talking about the zipDynamic3d test?